client/src/components/events/normal_events.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Text, Tag, Spacer} from 'cornell-glue-ui'
+import LikeContainer from './like'
+import { FlexContainer } from 'cornell-glue-ui'
+import { IEvent, IEventDate } from 'src/types/event.type'
+import { IUser } from 'src/types/user.type'
+import './event.css'
+
+
+
+interface EventProps {
+  event: IEvent, 
+  startTime: string
+  endTime: string
+}
+
+const Event: React.FC<EventProps> = ({event, startTime, endTime}) => {
```

Comment:
adding EventProps removes the type related errors here

![image](https://user-images.githubusercontent.com/32003815/147610483-81b69e8c-d1c1-435a-b2ac-121df84bdff1.png)